static/stylesheets/scss/directives/sk-tx-history.scss

```diff
@@ -0,0 +1,66 @@
+.tx-history-container {
+	.mt-20 {
+		margin-top: 20px;
+	}
+	.progress-bar {
+		margin-top: 10px;
+	}
+	.list {
+		> *  {
+			font-family: ProximaNovaRegular;
+		}
+		.list-item {
+			padding: 6px 0;
+			.view-transaction {
+				> * {
```

Comment:
This nesting is way too deep... Someone should rethink what class naming convention to be used and how messy you want to see your ``scss`` files.

Reply:
Again, this is out of scope for this PR, however it is something we're aware of and working towards as we modernise our UI implementation.